kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml

```diff
@@ -0,0 +1,30 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: bu-cs-jupyter-book
+  labels:
+    component.opendatahub.io/name: jupyterhub
+    opendatahub.io/component: 'true'
+    opendatahub.io/notebook-image: 'true'
+  annotations:
+    opendatahub.io/notebook-image-name:
+      "BU CS Jupyter Book"
+    opendatahub.io/notebook-image-url:
+      "https://github.com/aicoe-aiops/bu-cs-jupyter-book-notebook"
```

Comment:
I believe this is your url
'https://github.com/jappavoo/bu-cs-book-dev'

Reply:
Thanks!  Done